Resolved bug 1511: Components > Data Table > Enhancement as per Figma Design

Dipak7028 opened the pull request:
### _## Description_
> Resolved Components Data Table Enhancement as per Figma Design
1. Completed Enhancement Review Component as per the figma kit.

## Screenshots :
 
![image](https://github.com/user-attachments/assets/78d8a48d-2045-4b5e-8c6c-0a279748a253)
![image](https://github.com/user-attachments/assets/72ded938-755c-4b96-9e08-2433670947fa)

 
## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

Vikrantsingh441 reviewed (approved):
all changes reviewed and approved

rameshwarkamandar committed:
Resolved bug 1511: Components > Data Table > Enhancement as per Figma Design

Dipak7028 committed:
Resolved bug 1511: Components > Data Table > Enhancement as per Figma Design

Progress bar change
Badge alignment
Delete icon in center
Add one column is a badge

Dipak7028 committed:
Resolved bug 1511: Components > Data Table > Enhancement as per Figma Design

Progress bar change
Badge alignment
Delete icon in center
Add one column is a badge
Change As Per Rameshwar Branch

ashoksinarewai committed:
Merge branch 'development' into users/rameshwar/bug/1511-Components-Data-Table-Enhancement-as-per-Figma-Design

supriyartambewaiin committed:
Merge branch 'development' into users/rameshwar/bug/1511-Components-Data-Table-Enhancement-as-per-Figma-Design

rameshwarkamandar committed:
Merge branch 'development' into users/rameshwar/bug/1511-Components-Data-Table-Enhancement-as-per-Figma-Design

rameshwarkamandar committed:
Comment controls not needed design

ashoksinarewai reviewed (approved):
Approved
